service/src/test/java/uk/gov/hmcts/reform/fpl/service/DraftCMOServiceTest.java

```diff
@@ -83,15 +90,17 @@ void shouldReturnHearingDateDynamicListWhenCmoHasPreviousSelectedValue() {
     @Test
     void shouldReturnCaseManagementOrderWhenProvidedCaseDetails() {
         CaseDetails caseDetails = CaseDetails.builder()
-            .data(ImmutableMap.of("cmoHearingDateList", getDynamicList()))
-            .build();
+            .data(ImmutableMap.of("cmoHearingDateList", getDynamicList(), "allPartiesCustom",
+                createUnassignedDirection())).build();
```

Comment:
could we space this out a little nicer? 

ImmutableMap.of(
 "" , "",
 "", "",

and the build on new line